iOS_Concepts.playground/Contents.swift

```diff
@@ -1,4 +1,14 @@
 import UIKit
 
+// Mark-  Fibonacci Sequence Using Recursion
 
-var greeting = "Hello, playground"
+func FibonacciRecursion(numbSteps: Int, first: Int, second: Int)-> [Int]{
+    
+    if numbSteps == 0{
+        return[]
+    }
+    
+    return [first + second] + FibonacciRecursion(numbSteps: numbSteps - 1, first: second, second: first + second)
+}
+
+[0 , 1] + FibonacciRecursion(numbSteps: 4, first: 0, second: 1)
```

Comment:
This `[0 , 1] + ` function logic should be encapsulated within the function. Please follow OOP and SOLID principles always.